src/styles/loginCard.scss

```diff
@@ -1,9 +1,13 @@
+@import "./colours";
```

Comment:
These can also be removed, since we are already importing them in the `index.scss`